src/lesson8/competition/Participant.java

```diff
@@ -0,0 +1,18 @@
+package lesson8.competition;
+
+public abstract class  Participant {
+
+    public abstract String getName();
+    public abstract boolean getCondition();
+    public abstract void setCondition(boolean condition);
```

Comment:
no setters and getters need to be in the same class as variable that they return or set